README.md

```diff
@@ -37,7 +37,7 @@ The `ARA_API` module contains the core headers that define the actual ARA C API.
 `ARAInterface.h` is the main header, accompanied by a set of auxiliary headers for integrating ARA with
 each of the supported Companion APIs like `VST3` or `Audio Units`. Additionally, ARA Audio File Chunks
 are defined in `ARAAudioFileChunks.h`.
-It includes extensive printer-friendly [documentation](ARA_API/ARA_API.pdf).
```

Comment:
This is intentionally relative to its source!

Reply:
With this change, the link stops working in the local copy. We need a solution that works both online and locally, and I think that's something github will need to provide.